crypto/init.c

```diff
@@ -684,6 +679,11 @@ int OPENSSL_init_crypto(uint64_t opts, const OPENSSL_INIT_SETTINGS *settings)
             && !RUN_ONCE(&engine_dynamic, ossl_init_engine_dynamic))
         return 0;
 # ifndef OPENSSL_NO_STATIC_ENGINE
+#  if !defined(OPENSSL_NO_DEVCRYPTOENG)
```

Comment:
Here too

Reply:
Sorry, I guess I should have pushed the commit right away :disappointed:.  These were the only spots...

Reply:
Oh sorry, I thought you meant you had already pushed it.  Guess I'm excited :wink:

Reply:
I really admire your good humor.  I was just sure I was about to get scolded for wasting your time! :smiley:

Reply:
Nah, that's not the type of person I am.